client/src/components/Button/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import classes from './style.css'
+import {
+  faArrowLeft,
+  faArrowRight,
+  faCheckCircle
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+export default function Button(props) {
+  return (
+        <button className={classes.button} style={props.style} onClick={props.onClick}>
+          {props.leftIcon? <FontAwesomeIcon icon={faArrowLeft}/>: null}
+            <span>{props.children}</span>
```

Comment:
Add className for the span and use it in style 
don't use select element by tags

Reply:
Done